.github/workflows/linux.yml

```
@@ -0,0 +1,84 @@
+# Copyright (c) 2023 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under GPL v2 or later
+
+name: Build on Linux
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 3 * * 5'  # Every Friday at 3am
+  workflow_dispatch:
+
+jobs:
+  linux:
+    name: Build (${{ matrix.cc }})
```

```suggestion
    name: Build (${{ matrix.cc }} on ${{ matrix.runs-on }})
```

since we'll probably want to add more Linuxes here soon.

---

@mpartel it's important to note here that the only other Linux we can use for `runs-on` is Ubuntu 20.04, the LTS of yet another two years before, see https://github.com/actions/runner-images#available-images . We can cover the build on other distros with GitHub Actions but it will involve another layer, e.g. using Docker a la https://github.com/USBGuard/usbguard/blob/main/.github/workflows/docker.yml . With that said, do you still want me to make that addition?

---

@mpartel PS: added ` on ${{ matrix.runs-on }}` now as suggested.

---

Ok, I didn't realize the image selection was quite so limited, but let's keep it anyway now that you added it.